tests/integration/api_team_test.go

```diff
@@ -40,6 +41,7 @@ func TestAPITeam(t *testing.T) {
 	DecodeJSON(t, resp, &apiTeam)
 	assert.EqualValues(t, team.ID, apiTeam.ID)
 	assert.Equal(t, team.Name, apiTeam.Name)
+	assert.EqualValues(t, convert.ToOrganization(db.DefaultContext, org), apiTeam.Organization)
```

Comment:
```suggestion
	assert.EqualValues(t, convert.ToOrganization(db.DefaultContext, org, true), apiTeam.Organization)
```

Reply:
Or did I misunderstand something about the previous PR?

Reply:
There is no the third parameter for `convert.ToOrganization`. This function is helpt to convert `models.Organization` to `api.Organization`.

Reply:
Ah, that was `ToTeam`, not `ToOrganization`.